### PR TITLE
Update elasticsearch-xpack.yml

### PR DESCRIPTION
**Pull request by nduytg**
Refactor file module and fix file mode syntax to support octal number format

### CODE REVIEW

**Comment by jmlrt** [comment]
jenkins test this please

**Comment by jmlrt** [comment]
Hi @nduytg,
Thanks for your PR.
Can you sign the CLA: https://www.elastic.co/contributor-agreement

**Comment by nduytg** [comment]
Hi @jmlrt, yes I have signed it. Do I have to do anything else?

File: tasks/xpack/elasticsearch-xpack.yml [hunk 1]
```diff
@@ -8,4 +8,9 @@
 #Make sure elasticsearch.keystore has correct Permissions
 - name: Set elasticsearch.keystore Permissions
   become: yes
-  file: state=file path={{ es_conf_dir }}/elasticsearch.keystore owner=root group={{ es_group }} mode=0660
+  file: 
+    state: file
+    path: {{ es_conf_dir }}/elasticsearch.keystore
```
Comment (jmlrt) [inline tasks/xpack/elasticsearch-xpack.yml]:
```suggestion
    path: "{{ es_conf_dir }}/elasticsearch.keystore"
```

File: tasks/xpack/elasticsearch-xpack.yml [hunk 2]
```diff
@@ -8,4 +8,9 @@
...
+    owner: root
+    group: {{ es_group }}
```
Comment (jmlrt) [inline tasks/xpack/elasticsearch-xpack.yml]:
```suggestion
    group: "{{ es_group }}"
```

**Comment by jmlrt** [comment]
> tks master, i forgot to quote them :D

I often forget these quotes also, hopefully our CI tests are a good reminder :D

**Pull request merged**